test/fixture.py

```diff
@@ -242,7 +241,6 @@ def unstub_conf():
         content_access_cache_mock = MagicMock(name="ContentAccessCacheMock")
         inj.provide(inj.CONTENT_ACCESS_CACHE, content_access_cache_mock)
 
-        self.dbus_patcher = patch("subscription_manager.cli_command.cli.CliCommand._request_validity_check")
```

Comment:
When the `_request_validity_check` was removed, then it is also necessary to remove `self.dbus_patcher.start()` on following line. It fixes all unit test errors.